Sources/SwifterSwift/UIKit/UITableViewExtensions.swift

```diff
@@ -178,7 +178,10 @@ public extension UITableView {
     /// - Parameter indexPath: An IndexPath to check
     /// - Returns: Boolean value for valid or invalid IndexPath
     func isValidIndexPath(_ indexPath: IndexPath) -> Bool {
-        return indexPath.section < numberOfSections && indexPath.row < numberOfRows(inSection: indexPath.section)
+        return indexPath.section >= 0
```

Comment:
I'm more of the preference for having the `&&` at the end of the line, e.g.
```Swift
return indexPath.section >= 0 &&
   indexPath.row >= 0 &&
   indexPath.section < numberOfSections &&
   indexPath.row < numberOfRows(inSection: indexPath.section)
```
It's just a style thing, but I think it's a general standard. Thoughts @LucianoPAlmeida?

Reply:
Agreed, look nicer indeed :))